packages/block-library/src/list-item/edit.native.js

```diff
@@ -89,6 +89,12 @@ export default function ListItemEdit( {
 		renderAppender: false,
 	} );
 
+	// Add 0.62 hex opacity to baseColor text (e.g. #FFFFFF9E)
+	// or fallback to sensible default color
+	const defaultPlaceholderTextColor = style?.baseColors?.color?.text
+		? style.baseColors.color.text + '9E'
+		: '#87a6bc';
```

Comment:
What do you think if we move `#87a6bc` to the styles file? This would be helpful to add support for Dark mode as well.

We could use the [same values](https://github.com/WordPress/gutenberg/blob/trunk/packages/rich-text/src/component/style.native.scss#L14-L20) from the `RichText` component.

Reply:
Good call -- updated!